### ISSUE TITLE
Wrong number of classes exporting fine-tuned RF-DETR to ONNX

### ISSUE THREAD

**Issue by golden452** [roboflow/rf-detr#902]
### Search before asking

- [x] I have searched the RF-DETR issues and found no similar bug report.

### Bug

I fine tuned RFDETRLarge on a custom dataset with 8 classes. 

Default weight loading works just fine. the loaded model predicts 8 classes. 
Also double check shows that there are only 9 class names (plus 1 BACKGROUND.)

```
from rfdetr import RFDETRLarge

model = RFDETRLarge(pretrain_weights="/path/to/weight.pth")

class_names = model.class_names 
print(len(class_names))
```

But after exporting it to onnx, number of classes become 91 (90 from COCO, 1 background). prediction results contains 91 scores. 

```
# due to known issue, this code doesnt run in jupyter cells. put it in a standlone .py file and run
model = RFDETRLarge(pretrain_weights='path')

model.export(batch_size=5)
```

```
import onnxruntime as ort

# Load the ONNX model
session = ort.InferenceSession("output/inference_model.onnx")

# Get output information
for output in session.get_outputs():
    print(f"Output name: {output.name}")
    print(f"Output shape: {output.shape}")
    print(f"Output type: {output.type}") 
```

```
Output name: dets
Output shape: [5, 300, 4]
Output type: tensor(float)
Output name: labels
Output shape: [5, 300, 91]
Output type: tensor(float)
```

how should I deal with 91 label scores? do I just take the first 9?

Any help would be greatly appreciated. 

### Environment

RF-DETR 1.6.2
Ubuntu 24.04
Torch 2.11.0 + cu130

### Minimal Reproducible Example

model = RFDETRLarge(pretrain_weights='path')

model.export(batch_size=5)

### Are you willing to submit a PR?

- [ ] Yes, I'd like to help by submitting a PR!

**Comment by LeMinhNgan** [comment]
There is a hidden training parameter called `"num_classes"`, its default value is 90. You can change it if you want. It does not impact to prediction result via ONNX.
